FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -7,18 +7,28 @@ import Foundation
 public final class RemoteFeedLoader: FeedLoader {
 	private let url: URL
 	private let client: HTTPClient
+	private let mapper: FeedImageMapper
 
 	public enum Error: Swift.Error {
 		case connectivity
 		case invalidData
 	}
 
-	public init(url: URL, client: HTTPClient) {
+	public init(url: URL, client: HTTPClient, mapper: FeedImageMapper = RemoteFeedImageMapper.default) {
 		self.url = url
 		self.client = client
+		self.mapper = mapper
 	}
```

Comment:
Since there will be no need to create an instance of `RemoteFeedImageMapper` to call a `static` method, we can remove it.
```suggestion
	public init(url: URL, client: HTTPClient) {
		self.url = url
		self.client = client
	}
```